ui/pages/send/send-content/send-content.component.js

```diff
@@ -17,6 +18,10 @@ export default class SendContent extends Component {
     t: PropTypes.func,
   };
 
+  state = {
```

Comment:
Any reason to maintain this state here instead of in redux?

Reply:
I guess only that this state is used only within this component and the setter can be passed just one component down so felt unnecessary to kick up to redux but if its the preferred pattern happy to move it over!